config/schedule.rb

```diff
@@ -0,0 +1,28 @@
+# Use this file to easily define all of your cron jobs.
+#
+# It's helpful, but not entirely necessary to understand cron before proceeding.
+# http://en.wikipedia.org/wiki/Cron
+
+# Example:
+#
+set :output, "/home/deploy/simple-server/shared/logs/cron.log"
+#
+# every 2.hours do
+#   command "/usr/bin/some_great_command"
+#   runner "MyModel.some_method"
+#   rake "some:great:rake:task"
+# end
+#
+# every 4.days do
+#   runner "AnotherModel.prune_old_records"
+# end
+
+# Learn more: http://github.com/javan/whenever
+
+every :day, at: ['1:00 am'] do
```

Comment:
This will run at 1am UTC. Is that desired? Maybe we should put timezone in here.